moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_display.cpp

```diff
@@ -1184,9 +1184,15 @@ void MotionPlanningDisplay::onRobotModelLoaded()
 
   dynamics_solver_.clear();
   for (std::size_t i = 0; i < groups.size(); ++i)
-    if (getRobotModel()->getJointModelGroup(groups[i])->isChain())
-      dynamics_solver_[groups[i]].reset(
-          new dynamics_solver::DynamicsSolver(getRobotModel(), groups[i], gravity_vector));
+  {
```

Comment:
Is this parenthesis required?

Reply:
This parenthesis is required. I think this source corresponds to "when the block is more complicated" of the ROS code style (http://wiki.ros.org/ja/CppStyleGuide#A.2BMLMw.2FDDJMNUwqTD8MN4wwzDI-).

Reply:
OK.